parl/remote/utils.py

```diff
@@ -52,49 +54,115 @@ def data_process():
     return to_write_lines
 
 
-def load_remote_class(file_name, class_name, end_of_file, in_sys_path):
+def is_class_defined_in_file(cls):
```

Comment:
is_class_defined_in_file -> is_implemented_in_notebook.

and update the comment: 
check if the remote class is implemented in notebook environment.

Reply:
check if the remote class is implemented in the environments like notebook(e.g., ipython, notebook).

Reply:
fixed